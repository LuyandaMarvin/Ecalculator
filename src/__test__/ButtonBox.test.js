import React from "react";
import ButtonBox from "../components/ButtonBox"
import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

configure({ adapter: new Adapter() });


describe('<ButtonBox />', () => {
  const buttonBox = shallow(<ButtonBox />);
  
    it('renders the correct html output', () => {
      expect(buttonBox.html()).toMatchSnapshot();
    });
});