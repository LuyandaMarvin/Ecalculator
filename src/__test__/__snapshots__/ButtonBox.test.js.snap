// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ButtonBox /> renders the correct html output 1`] = `"<div class=\\"buttonBox\\"></div>"`;
